mlflow_faculty/mlflow_converters.py

```diff
@@ -40,18 +40,25 @@
 from mlflow.utils.mlflow_tags import MLFLOW_RUN_NAME, MLFLOW_PARENT_RUN_ID
 
 _FACULTY_TO_MLFLOW_RUN_STATUS_MAP = {
-    FacultyExperimentRunStatus.RUNNING: RunStatus.RUNNING,
-    FacultyExperimentRunStatus.FINISHED: RunStatus.FINISHED,
-    FacultyExperimentRunStatus.FAILED: RunStatus.FAILED,
-    FacultyExperimentRunStatus.SCHEDULED: RunStatus.SCHEDULED,
+    FacultyExperimentRunStatus.RUNNING: "RUNNING",
+    FacultyExperimentRunStatus.FINISHED: "FINISHED",
+    FacultyExperimentRunStatus.FAILED: "FAILED",
+    FacultyExperimentRunStatus.SCHEDULED: "SCHEDULED",
+    FacultyExperimentRunStatus.KILLED: "KILLED",
 }
 
 
 _MLFLOW_TO_FACULTY_RUN_STATUS_MAP = {
```

Comment:
Maybe include a comment about why we're mapping both strings and mlflow `RunStatus` objects?